<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="20845" started-at="2017-08-02T10:35:25Z" finished-at="2017-08-02T10:35:46Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="20845" started-at="2017-08-02T10:35:25Z" finished-at="2017-08-02T10:35:46Z">
      <class name="Suite.Suiteclass">
        <test-method status="PASS" signature="Startsuite()[pri:0, instance:Suite.Suiteclass@482f8f11]" name="Startsuite" is-config="true" duration-ms="31" started-at="2017-08-02T16:05:25Z" finished-at="2017-08-02T16:05:25Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Startsuite -->
        <test-method status="PASS" signature="browserInitiate(java.lang.String)[pri:0, instance:Suite.Suiteclass@482f8f11]" name="browserInitiate" is-config="true" duration-ms="20173" started-at="2017-08-02T16:05:25Z" finished-at="2017-08-02T16:05:46Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[ff]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- browserInitiate -->
        <test-method status="FAIL" signature="M()[pri:0, instance:Suite.Suiteclass@482f8f11]" name="M" duration-ms="32" started-at="2017-08-02T16:05:46Z" finished-at="2017-08-02T16:05:46Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at Module.Loginpage.Sigin(Loginpage.java:17)
	at Suite.Suiteclass.M(Suiteclass.java:23)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- M -->
      </class> <!-- Suite.Suiteclass -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
